src/pages/library/DataBrowserPreview.js

```diff
@@ -108,7 +138,18 @@ const DataBrowserPreview = ({ id }) => {
           columnSettings.length > 0 && h(ColumnSelector, { onSave: setColumnSettings, columnSettings })
         ]),
         previewData?.length === 0 && div({ style: { width: '100%', height: '100%', display: 'flex', alignItems: 'center', justifyContent: 'center' } }, ['(No Data)'])
-      ])
+      ]),
+    viewJSON && h(ModalDrawer, {
```

Comment:
Could we add a title to the drawer? Perhaps something like "Viewing JSON for version _verionID_"

Reply:
I added a title that is `(tablename) row (row#) - (column name)` - im just not sure that version will be a field available for all data types! Please let me know if you think this is okay!

Reply:
That works. Based on our conversation with Lou I think we will be iterating on this. But having a title is a good cue for the user to let them know which JSON they are looking at. Thanks!